synapse/tests/test_lib_storm.py

```diff
@@ -297,28 +268,12 @@ class StormTest(s_t_utils.SynTest):
             self.len(1, nodes)
             self.eq(0x05060708, nodes[0].ndef[1])
 
-            # Sad paths where there are no nodes which match the specified values.
-            self.len(0, await core.nodes('test:guid | max :newp'))
-            self.len(0, await core.nodes('test:guid | min :newp'))
-
-            # Sad path for a form, not a property; and does not exist at all
-            with self.raises(s_exc.BadSyntax):
-                await core.nodes('test:guid | max test:newp')
-            with self.raises(s_exc.BadSyntax):
-                await core.nodes('test:guid | min test:newp')
-
-            # Ensure that max evaluates ival properties as the upper bound.
-            async with await core.snap() as snap:
-                node = await snap.addNode('test:guid', '*', {'tick': '2015',
-                                                             '.seen': (minval, maxval)})
-                await node.addTag('maxtest')
-                node = await snap.addNode('test:guid', '*', {'tick': '2016',
-                                                             '.seen': (midval, midval + 1)})
-                await node.addTag('maxtest')
+            # Sad paths where the specify an invalid property name
+            with self.raises(s_exc.NoSuchProp):
+                self.len(0, await core.nodes('test:guid | max :newp'))
```

Comment:
this is a breaking change we'll need to doc

Reply:
Not sure how important it will be that it now raises NoSuchProp rather than InvalidSyntax, but we can mention it i suppose.